### ISSUE TITLE
SimBA attack parameters for CIFAR 10

### ISSUE THREAD

**Issue by amir1m** [Trusted-AI/adversarial-robustness-toolbox#1157]
Hello,
I am running SimBA un-targeted attack on CIFAR 10 CNN classifier(logit). I am getting very low success rate of 1% with iterations=10000 and freq_dim=32. Are there any specific parameters to be used?

Thanks.

**Comment by beat-buesser** [comment]
Hi @amir1m Thank you very much for using ART!

Would you have a code example in a Google Colab notebook that reproduces you experiment?

**Comment by amir1m** [comment]
Hi @beat-buesser,
Thanks for your response!

Link to the Notebook: https://gist.github.com/amir1m/44f9a015eb74157da645dd9f7a69c89b

Thanks.

**Comment by beat-buesser** [comment]
Hi @amir1m I think a main issue is that your test labels `test_labels` are index labels but in `get_random_correct_samples` you treat them as one-hot-encoded. That way' `correct_indices` does not describe the correctly classified samples, but samples of class `0`. The attack `SimBa` works according to the paper on predicted probabilities, therefore I would add a softmax activation to you model, but I'm not sure how much of a difference it will make.
